src/app/pages/home/home.component.html

```diff
@@ -40,8 +40,10 @@ <h2 class="mb-3 px-5 home-text header-text">What is virtrolio?</h2>
               <span>Sign in with Google</span>
             </button>
           </div>
-          <p class="lead pt-4" id="disclaimer-text">By signing up for our service, you are agreeing to our <a
-            routerLink="/terms-of-service">Terms of Service</a>.</p>
+          <p class="lead pt-4" id="disclaimer-text">By signing up for our service, you are agreeing to our
+            <a href="/terms-of-service">Terms of Service.</a><br> We currently do not support other sign-in
+            providers.
+          </p>
```

Comment:
Stop using `<br>` >:(
```suggestion
            <a href="/terms-of-service">Terms of Service.</a>
          </p>
          <p>At this time, we do not support signing in using providers other than Google.</p>
```